src/main/java/core/basesyntax/FiguresAreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FiguresAreaCalculator {

```

Comment:
Do not mix interface and implementation
```suggestion
public interface AreaCalculator {

```